dialog-el.html

```diff
@@ -74,7 +77,8 @@
       left: 50%;
       transform: rotate(45deg) translate3d(-50%,-5px,0);
       z-index: -1;
-      background-color: var(--dialog-el-background, #FFFFFF);
+      background-color: #FFFFFF;
+      background-color: var(--dialog-el-background);
```

Comment:
What is the benefit of having the fallback value there as well as in the line above it?